web/src/extract/extension.rs

```diff
@@ -49,7 +49,7 @@ impl Deref for ExtensionsRef<'_> {
 impl<'a, 'r, 's, S> FromRequest<'a, &'r mut WebRequest<'s, S>> for ExtensionsRef<'a> {
     type Type<'b> = ExtensionsRef<'b>;
     type Error = Infallible;
-    type Future = impl Future<Output = Result<Self, Self::Error>>;
+    type Future = impl Future<Output = Result<Self, Self::Error>> where &'r mut WebRequest<'s, S>: 'a;
```

Comment:
could've used `where 'r: 'a` instead. It's less verbose but might seem unintuitive. so?

Reply:
this is fine for internal use.